cktap/proto.py

```diff
@@ -204,6 +205,23 @@ def get_xpub(self, cvc, master=False):
         xpub = st['xpub']
         return encode_base58_checksum(xpub)
 
+    def get_xpub_at_path(self, cvc, bip32_path: str):
```

Comment:
`get_xpub_at_path` is not the best name: there is a side effect here: the card's derivation is changed... so not a getter